tool/cli.js

```diff
@@ -46,11 +46,43 @@ program
   .name("tool")
   .version("0.0.0")
   .disableGlobalOption("--silent")
-  .command("validate-redirects", "Check the _redirects.txt file(s)")
+  .cast(false)
+  .command("validate-redirects", "Try loading the _redirects.txt file(s)")
+  .argument("[locales...]", "Locale", {
+    default: [...VALID_LOCALES.keys()],
+    validator: [...VALID_LOCALES.keys()],
+  })
+  .option("--strict", "Strict validation")
```

Comment:
I appreciate that the CI needs to change to run `validate-redirects --strict` now. 
But when would you run this CLI *without* the `--strict`?

Reply:
This question is the last time remaining. And once this lands you'll update the `pr-test.yml` in the mdn/content repo, right?

Reply:
The idea was to keep the `validate-redirects` for yari CI and use `validate-redirects --strict` for `(translated-)content`

Reply:
When would you run `validate-redirects` in Yari? Just for (manually) sanity checking periodically?

Reply:
Keep it like it is. `validate-redirects` still assures that redirects can be loaded.